### PR TITLE
Update documentation for cluster

### PR DESCRIPTION
**Pull request by khooroko**

### CODE REVIEW

**Review by jelneo — APPROVED** [review]
LGTM

**Review by lawwman — APPROVED** [review]
Explanation is clear. Approved

File: docs/UserGuide.adoc [hunk 1]
```diff
@@ -23,7 +23,8 @@ The terms used in this guide are as follows: +
 
 * Blacklist - List of people who are banned +
 * Whitelist - List of people who have cleared their debts +
-* Index - The ordering of a contact in the list
+* Index - The ordering of a contact in the list +
+* Cluster - The general location of a person based on [https://www.ura.gov.sg/realEstateIIWeb/resources/misc/list_of_postal_districts.htm[postal districts]]
```
Comment (jaivigneshvenugopal) [inline docs/UserGuide.adoc:28]: Good that you included the link!

**Review by jaivigneshvenugopal — APPROVED** [review]
Looks good man!

**Pull request merged**